CI.rst

```diff
@@ -529,65 +404,60 @@ For more information, see:
 
 Website endpoint: http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/
 
-Naming conventions for stored images
-====================================
 
-The images produced during the ``Build Images`` workflow of CI jobs are stored in the
-`GitHub Container Registry <https://github.com/orgs/apache/packages?repo_name=airflow>`_
+Debugging CI Jobs in Github Actions
```

Comment:
Added chapter on how to debug CI jobs - all the possibilities of how you can test different parts are now nicely separated out.